### PR TITLE
remove helm retry (eks unauthorized error fixed)

### PR DESCRIPTION
**Pull request by prekoa**

### CODE REVIEW

File: helm/kubernetes.go [hunk 1]
```diff
@@ -34,13 +33,16 @@ var tillerTunnel *kube.Tunnel
...
 func GetK8sConnection(kubeConfig []byte) (*kubernetes.Clientset, error) {
+	var client *kubernetes.Clientset
 	config, err := GetK8sClientConfig(kubeConfig)
 	if err != nil {
 		return nil, fmt.Errorf("create kubernetes config failed: %v", err)
 	}
-	client, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		return nil, fmt.Errorf("create kubernetes connection failed: %v", err)
+	for i := 0; i < 2; i++ {
```
Comment (baluchicken) [inline helm/kubernetes.go]: please write a comment why we need to get the client for two times without an error.

File: cluster/hooks.go [hunk 2]
```diff
@@ -32,7 +32,9 @@ import (
 	"github.com/banzaicloud/pipeline/internal/security"
 	pkgCluster "github.com/banzaicloud/pipeline/pkg/cluster"
 	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
+	helm2 "github.com/banzaicloud/pipeline/pkg/helm"
```
Comment (baluchicken) [inline cluster/hooks.go]: this import is present as pkgHelm so it is unnecessary to import as helm2 again

File: cluster/hooks.go [hunk 3]
```diff
@@ -32,7 +32,9 @@ import (
...
 	pkgHelm "github.com/banzaicloud/pipeline/pkg/helm"
```
Comment (baluchicken) [inline cluster/hooks.go]: maybe we should rename this to helm2 because in every other place this import called that way

Comment (prekoa) [reply cluster/hooks.go]: thx, fixed

**Pull request merged**